### PR TITLE
very begining fix commit

### PR DESCRIPTION
**Pull request by Sarutoh**
i am going to fix issue when updating only one  transfer record(but 2 expected)

### CODE REVIEW

**Comment by sveredyuk** [comment]
hi @Sarutoh. Very strange change. Only 1 new file and no usage of this file )

**Comment by Sarutoh** [comment]
hi @sveredyuk . sorry, forgot to create new branch, started codding on main :(. that`s just  a draft to try create pull request

File: spec/services/record/update_spec.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
```
Comment (sveredyuk) [inline spec/services/record/update_spec.rb]: You have app/services/update_record_service.rb

So spec should be also located as spec/services/update_record_service_spec.rb

**Review by sveredyuk — COMMENTED** [review]
👍

**Pull request merged**